src/lib/__tests__/server-utilities.test.js

```diff
@@ -38,16 +40,31 @@ describe('md5Hash', () => {
 
 describe('compressJPEG', () => {
   if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
-  fs.closeSync(fs.openSync('./tmp/test.jpg', 'a')); // Create empty file 'test.jpg'
   const filePath = './tmp/test.jpg';
 
-  it('overwrites uncompressed image with compressed image', () => {
+  // Copy compressible JPEG to `./tmp` directory to mimick JPEG upload
+  copyFile('./static/test-assets/compression-test.jpg', filePath);
+
+  /* Not all files will compress */
+  it('compresses compressible JPEG', () => {
+    const uncompressedSize = fs.statSync(filePath).size;
+    compressJPEG(filePath).then(() => {
+      const compressedSize = fs.statSync(filePath).size;
+      expect(uncompressedSize).toBeGreaterThan(compressedSize);
+    });
+  });
+
+  /* `./tmp/test.jpg` is already compressed from previous test */
+  it("doesn't recompress already compressed JPEG", () => {
+    const oldCompressedSize = fs.statSync(filePath).size;
     compressJPEG(filePath).then(() => {
-      expect(fs.existsSync(filePath)).toBe(true);
+      const newCompressedSize = fs.statSync(filePath).size;
+      expect(newCompressedSize).toEqual(oldCompressedSize);
     });
   });
 
-  it('deletes image', () => {
+  /* Clears `./tmp` directory after compression and upload to S3 */
+  it('deletes JPEG', () => {
```

Comment:
Use afterAll() here and return a promise so it knows it's asynchronous. 

That's if you're simply doing cleanup anyway. If you want to add a test for deletion working you should probably put it in a different `describe` etc.